web/static/js/services/retro_channel.js

```diff
@@ -60,6 +61,13 @@ const applyListenerCallbacks = (retroChannel, store, actions) => {
     })
   })
 
+  retroChannel.on("user_typing_action_item", ({ userToken }) => {
```

Comment:
see lines 57-62, as this handler's content is verbatim with the exception of the event name

Reply:
hmmm... we gotta do something about this duplication